tests/chart_tests/test_nginx_service.py

```diff
@@ -262,3 +262,48 @@ def test_nginx_disable_leader_election_overrides(self):
         c_by_name = get_containers_by_name(doc)
         disableLeaderElection = "--disable-leader-election=true"
         assert disableLeaderElection in c_by_name["nginx"]["args"]
+
+    def test_nginx_with_capability_default(self):
```

Comment:
test_nginx_with_capability_default -> test_nginx_with_capability_defaults

Reply:
we already have a default function called def test_nginx_defaults(self) can you add there itself

Reply:
done